tslint.json

```diff
@@ -0,0 +1,100 @@
+{
+    "defaultSeverity": "error",
+    "extends": "typestrict",
+    "jsRules": {},
+    "rules": {
+        "arrow-parens": true,
+        "ban-ts-ignore": true,
+        "import-spacing": true,
+        "indent": {
+            "options": [
+                "spaces",
+                2
+            ]
+        },
+        "max-line-length": {
+            "options": {
+                "limit": 120
+            }
+        },
+        "no-debugger": true,
+        "no-console": true,
+        "new-parens": true,
+        "only-arrow-functions": {
+            "options": [
+                "allow-declarations",
+                "allow-named-functions"
+            ]
+        },
+        "no-consecutive-blank-lines": [
+            true,
+            2
+        ],
+        "no-irregular-whitespace": true,
+        "no-trailing-whitespace": {
+            "options": [
+                "ignore-template-strings"
+            ]
+        },
+        "no-unused-variable": false, // disabled because deprecated
+        "number-literal-format": true,
+        "quotemark": {
+            "options": [
+                "single",
```

Comment:
It looks like double-quote is the most common quote used in this project. I wanted to suggest using single-quotes simply because they are less noisy and easier to parse when next to each other in my opinion.